src/test/scala/org/scalatestplus/junit5/ScalaTestEngineSpec.scala

```diff
@@ -0,0 +1,65 @@
+package org.scalatestplus.junit5
+
+import org.junit.platform.engine.UniqueId
+import org.junit.platform.engine.discovery.ClasspathRootSelector
+import org.junit.platform.engine.discovery.DiscoverySelectors.{selectClasspathRoots, selectPackage}
+import org.junit.platform.launcher.core.LauncherDiscoveryRequestBuilder.request
+import org.scalatest.{BeforeAndAfterAll, funspec}
+import org.scalatestplus.junit5.helpers.HappySuite
+
+import java.nio.file.{Files, Paths}
+import scala.jdk.CollectionConverters.SetHasAsScala
```

Comment:
May change to use the old: 

```
import scala.collection.JavaConverters._ 
```
here since we still need to support old Scala versions.  You may try: 

```
> sbt +test
```

to see if all works correctly.

Reply:
Fixed, thanks for giving me the command. I am not well versed in scala ecosystem so apologies for the extra rounds of review.